meteor/client/ui/DndListWrapper.tsx

```diff
@@ -0,0 +1,66 @@
+import { DropTargetMonitor, useDrop } from 'react-dnd'
+import { DragDropItemTypes } from './DragDropItemTypes'
+import React, { useEffect, useState } from 'react'
+import update from 'immutability-helper'
+import { DndListItemWrapper } from './DndListItemWrapper'
+
+interface IDndListWrapperProps {
+	className: string
+	dndType: DragDropItemTypes
+	stateUIArray: any[]
+
+	onDrop: (newArray: any[]) => void
+	provideItemForRender: (item: any, index: number) => JSX.Element
```

Comment:
I see how you came to this name, but you can just call it `renderItem` or `itemRenderer`

Reply:
Changed!